### PR TITLE
feat: add local typings

### PR DESCRIPTION
**Pull request by damusix**

### CODE REVIEW

**Review by devinstewart — APPROVED** [review]
Looks nice

File: lib/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+        toHex: (number: number) => string;
+    }
+}
+
+// declare const hapiAuthJwt: {
+//     plugin: Plugin<void>;
+//     token: hapiJwt.Token;
+//     crypto: hapiJwt.Crypto;
+//     utils: hapiJwt.Utils;
+// };
```
Comment (Marsup) [inline lib/index.d.ts:451]: Why keep this part?

**Review by Marsup — APPROVED** [review]
Do you know if it's backward compatible with previous types or did you experience some breaking changes?

File: lib/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+        | 'RS512'
+        | 'PS256'
+        | 'PS384'
+        | 'PS512'
+        | 'ES256'
+        | 'ES384'
+        | 'ES512'
+        | 'HS256'
+        | 'HS384'
+        | 'HS512';
```
Comment (mautematico) [inline lib/index.d.ts:37]: see #53, EdDSA supported as well 😄

Comment (Marsup) [inline lib/index.d.ts:37]: As already mentioned:
```suggestion
        | 'HS512'
        | 'EdDSA';
```

File: lib/index.d.ts (hunk 1 above)
Comment (damusix) [reply lib/index.d.ts]: removed

**Pull request merged**